### PR TITLE
feat: ispreinstalled flag

### PR DESCRIPTION
**Pull request by ElderMatt**
This PR adds isPreInstalled flag for the installer.
See: [APL-134](https://jira.linode.com/browse/APL-134)

Also removed Minio dependency for everything

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -86,7 +90,14 @@ on:
           - letsencrypt_staging
           - letsencrypt_production
         default: letsencrypt_production
-
+      is_pre_installed:
```
Comment (CasLubbers) [inline .github/workflows/integration.yml:93]: I would add this as an `install_profile`, something like `minimal-pre-installed`. And maybe set it as the default?

File: .github/workflows/integration.yml [hunk 2]
```diff
@@ -86,7 +90,14 @@ on:
...
+        type: choice
+        description: Fake if Otomi is pre-installed by Installer
```
Comment (CasLubbers) [inline .github/workflows/integration.yml:95]: Why `Fake`? Do you mean `False`?

File: .github/workflows/integration.yml [hunk 3]
```diff
@@ -274,12 +286,29 @@ jobs:
           CLIENT_SECRET: ${{ secrets.EDGEDNS_CLIENT_SECRET }}
         run: |
           touch values.yaml
+          [[ '${{ inputs.dns }}' == 'az_dns' ]] && echo "$AZ_DNS" >> values.yaml && domainSuffix="--set cluster.domainSuffix=tst-${{ github.run_id }}.aks.redkubes.net"
```
Comment (CasLubbers) [inline .github/workflows/integration.yml:289]: Are these needed? I think we do not use azure anymore right?

**Review by CasLubbers — DISMISSED** [review]

File: .github/workflows/integration.yml (hunk 2 above)
Comment (ElderMatt) [reply .github/workflows/integration.yml:95]: This flag is supposed to be set by the APL-Installer, I added this option as a way to fake it for easier testing.

File: .github/workflows/integration.yml (hunk 3 above)
Comment (ElderMatt) [reply .github/workflows/integration.yml:289]: Good find, I think this because a merge conflict.

File: .github/workflows/integration.yml (hunk 2 above)
Comment (CasLubbers) [reply .github/workflows/integration.yml:95]: Oh I read the sentence completely wrong, apologies.

File: .github/workflows/integration.yml (hunk 1 above)
Comment (ElderMatt) [reply .github/workflows/integration.yml:93]: would this in addition to a workflow parameter or instead of the workflow parameter?

**Review by ferruhcihan — APPROVED** [review]
LGTM

**Pull request merged**